src/components/ChangelogSnippet/ChangelogCard.module.css

```diff
@@ -1,27 +1,40 @@
-/* Card Wrapper */
+/* Card Wrapper used in Changelog page */
```

Comment:
Mobile responsiveness missing. it should look like this: 
<img width="490" height="253" alt="image" src="https://github.com/user-attachments/assets/924316d7-31d4-42f9-a690-2b8896b7ae9b" />